### PR TITLE
Re-organize what's new for 2.0

### PR DESCRIPTION
**Pull request by astrofrog**
**This is a work in progress - don't merge!**

This is a reorganization of the 'what's new' for 2.0. I've tried to re-order things so that bigger features come first, and I've also combined some of the existing sections together to avoid having too many individual sections. You can see a preview of the current version of this branch here:

http://astrofrog-debug.readthedocs.io/en/latest/whatsnew/2.0.html

There are several things that need to be resolved:

* [x] Three sections at the bottom are marked as **REMOVE**  because I don't think they are big enough features/changes to warrant being listed here. @nden @pllim @crawfordsm - since these are modeling and stats related, this is your chance to object

* [x] I think it would be nice to show an actual example for the velocity stuff, either for the experimental stuff or the barycentric etc correction method on SkyCoord - @eteq @adrn 

* [ ] The Ripley K function is not very self-explanatory (for a start I don't know what that function is and there are just a set of curves in the plot). Is there any way to make the example simpler in code and more verbose in text? @crawfordsm 

I propose that changes are made via PRs to my branch so that we can preview all the changes before merging it in and doing the release.

### CODE REVIEW

File: docs/whatsnew/2.0.rst [hunk 1]
```diff
@@ -249,26 +236,88 @@ to atomic mass unit (amu). For example::
...
+<mixin_columns>` types in an ASCII :ref:`ECSV
+<ecsv_format>` table file.  The table file can then be read back
+into astropy with no loss of object data or attributes.
 
-    >>> from astropy import units as u
-    >>> from astropy.modeling.models import Gaussian1D
-    >>> g = Gaussian1D(amplitude=1*u.J, mean=1*u.m, stddev=0.1*u.m)
-    >>> g([3, 4, 5.5] * u.cm)
-    <Quantity [  3.70353198e-21,  9.72098502e-21,  4.05703276e-20] J>
+- `astropy.time.Time`
```
Comment (bsipocz) [inline docs/whatsnew/2.0.rst]: this list looks rather odd in the rendered version. Wouldn't it be enough just to list the short names in a list after each other?

Comment (astrofrog) [inline docs/whatsnew/2.0.rst]: I agree, that was bugging me too.

Comment (astrofrog) [inline docs/whatsnew/2.0.rst]: Done

File: docs/whatsnew/2.0.rst [hunk 2]
```diff
@@ -249,26 +236,87 @@ to atomic mass unit (amu). For example::
...
+---------------
 
-.. _whatsnew-2.0-n-submodels:
+The SAMP (Simple Application Messaging Protocol) module, formerly available in
+``astropy.vo.samp``, has now been moved to ``astropy.samp``, so you should
+update any imports to this module.
 
-New ``n_submodels`` shared method in single and compound models
-===============================================================
+REMOVE? - New ``n_submodels`` shared method in single and compound models
```
Comment (pllim) [inline docs/whatsnew/2.0.rst]: I am neutral on removing this and the other entry below. @nden should make the final decision as she is the `modeling` maintainer.

File: docs/whatsnew/2.0.rst [hunk 3]
```diff
@@ -300,76 +346,15 @@ For example::
     >>> p(x, with_bounding_box=True, fill_value=-100)
     array([-100., -100., -100., -100.,   -1.,    2.,    5.,    8.,   11.,   14.])
 
+REMOVE - New ``std_ddof`` keyword to :func:`~astropy.stats.sigma_clipped_stats`
```
Comment (larrybradley) [inline docs/whatsnew/2.0.rst]: Feel free to remove this section.  I added it only because it was requested.

File: docs/whatsnew/2.0.rst (hunk 2 above)
Comment (nden) [inline docs/whatsnew/2.0.rst]: I don't object removing this one and the one below.

**Comment by eteq** [comment]
See astrofrog/astropy#68 for part of what you asked from for @adrn and I (@adrn I think is going to send you something with a velocity frame transformation example separately?)

**Comment by adrn** [comment]
Done!

**Comment by astrofrog** [comment]
@mwcraig - if you have a chance, could you add some links in the CCDData section? In particular, maybe CCDData and nddata should be API links? If you don't have time I'll do this in a few hours.

**Comment by astrofrog** [comment]
Well at least this is in a mergeable state so will go ahead and merge it now, and if anyone wants to improve the links e.g. in the CCD section, please go ahead (I have to be offline for a few hours)

**Comment by mwcraig** [comment]
@astrofrog  - Sorry, won't be able to get it done until a few hours from now...

**Pull request merged**